docker-compose.yml

```diff
@@ -16,6 +17,13 @@ services:
             placement:
                 constraints:
                     - node.role == worker
+            update_config:
+              parallelism: 1
+              delay: 10s
+              failure_action: rollback
```

Comment:
While I would like to use rollback, it doesn't look like its supported by the version of docker on our servers. We will have to use "pause" for now.